examples/offloading-multiple-pods/manifests/cluster.yaml

```
@@ -0,0 +1,8 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+networking:
+  serviceSubnet: "10.111.0.0/16"
+  podSubnet: "10.112.0.0/16"
+nodes:
+  - role: control-plane
+    image: kindest/node:v1.25.0
```

Is there any reason to use different configurations compared to the other examples?

```suggestion
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
networking:
  serviceSubnet: "10.90.0.0/12"
  podSubnet: "10.200.0.0/16"
nodes:
  - role: control-plane
    image: kindest/node:v1.23.6
  - role: worker
    image: kindest/node:v1.23.6
```

---

I preferred to use a single-node cluster to make the example lighter since you need to create 3 clusters (like in https://github.com/cheina97/liqo/blob/frc/exampleMultiplePodOffloading/examples/offloading-with-policies/manifests/cluster.yaml)

---

Fine, but use the same version and subnets of the other examples 👍